src/modules/blockchain/implementation/web3-service.js

```diff
@@ -973,6 +973,12 @@ class Web3Service {
         };
     }
 
+    getMaxNumberOfHistoricalBlocksForSync() {
+        return Math.round(
```

Comment:
Maybe calculate this and put in some const value so it's not calculated every thime